x-pack/plugins/apm/public/components/alerting/ui_components/alert_details_app_section/alert_details_app_section.tsx

```diff
@@ -59,7 +60,47 @@ export function AlertDetailsAppSection({
   rule,
   alert,
   timeZone,
+  setAlertSummaryFields,
 }: AlertDetailsAppSectionProps) {
+  useEffect(() => {
+    const alertSummaryFields = [
+      {
+        label: (
+          <FormattedMessage
+            id="xpack.apm.pages.alertDetails.alertSummary.actualValue"
+            defaultMessage="Actual value"
+          />
+        ),
+        value: formatAlertEvaluationValue(
+          alert?.fields[ALERT_RULE_TYPE_ID],
+          alert?.fields[ALERT_EVALUATION_VALUE]
+        ),
+      },
+      {
+        label: (
+          <FormattedMessage
+            id="xpack.apm.pages.alertDetails.alertSummary.expectedValue"
+            defaultMessage="Expected value"
+          />
+        ),
+        value: formatAlertEvaluationValue(
+          alert?.fields[ALERT_RULE_TYPE_ID],
+          alert?.fields[ALERT_EVALUATION_THRESHOLD]
+        ),
+      },
+      {
+        label: (
+          <FormattedMessage
+            id="xpack.apm.pages.alertDetails.alertSummary.source"
+            defaultMessage="Source"
+          />
+        ),
+        value: '-',
+      },
```

Comment:
Fields I'd expect to see for all APM rule details (maybe they are already there): `service.name`, `service.environment`.

For transaction rules I'd additionally expect: `transaction.type` (and if available `transaction.name`).
For error rules I'd additionally expect: `error.grouping_key`.